redis/jobs/consumers/consumer_scan_mod_on_virus_total.go

```diff
@@ -40,7 +40,12 @@ func ScanModOnVirusTotalConsumer(ctx context.Context, payload []byte) error {
 	log.Ctx(ctx).Info().Msgf("starting virus scan of mod %s version %s", task.ModID, task.VersionID)
 
 	version := postgres.GetVersion(ctx, task.VersionID)
-	link := storage.GenerateDownloadLink(version.Key)
+	mod := postgres.GetModByID(ctx, version.ModID)
+
+	//link := storage.GenerateDownloadLink(version.Key)
```

Comment:
Please remove the unused code

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/8ea4f2b30882e41af00e95ce37ab41b3a4ebd4e3